Classes/Issues/Merge/IssueMergeContextCell.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  IssueMergeContextCell.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 2/11/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+import SnapKit
+import SDWebImage
+
+final class IssueMergeContextCell: IssueCommentBaseCell, ListBindable {
+
+    private let avatarView = UIImageView()
+    private let iconView = UIImageView()
+    private let titleLabel = UILabel()
+    private let descriptionLabel = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        contentView.addSubview(avatarView)
+        contentView.addSubview(iconView)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(descriptionLabel)
+
+        avatarView.clipsToBounds = true
```

Comment:
Can we add the ignore invert colors on this view?

```swift
if #available(iOS 11, *) {
  avatarView.accessibilityIgnoresInvertColors = true
}
```

Reply:
Sounds good